xc/xenops_server_xen.ml

```
@@ -201,6 +201,12 @@ let safe_rm xs path =
     xs.Xs.rm path
   with _ -> ()
 
+let safe_write xs path value =
+  debug "xenstore-write %s <- %s" path value;
+  try
+    xs.Xs.write path value
+  with _ -> ()
```

Why do you ignore errors here?